test/letters/actions/letters.unit.spec.js

```diff
@@ -117,6 +117,8 @@ describe('saveAddress', () => {
     const dispatch = sinon.spy();
     thunk(dispatch, getState)
       .then(() => {
+        throw new Error('Should not resolve the promise when GET fails');
```

Comment:
If I understand correctly, this is necessary to mimic the nested structure of the new consolidated action creator